scripts/ansible/roles/golang_base/tasks/centos-8.yml

```diff
@@ -0,0 +1,7 @@
+- name: Install Go
+  become: true
+  yum:
+    name:
+      - golang  # 1.13.4 in July 2020
```

Comment:
Same here?

Reply:
On CentOS 8 golang is packaged with the OS, so the version is still 1.13.x, specifially [golang-1.13.15-1.module_el8.2.0](http://mirror.centos.org/centos/8/AppStream/x86_64/os/Packages/golang-1.13.15-1.module_el8.2.0+490+5bc58d45.x86_64.rpm)